index.html

```diff
@@ -1,248 +1,330 @@
-                <li>Capybara</li>
-                <li>Selenium</li>
-            </ul>
-        </div>
+        <ul class="portfolio-languages">
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
 
-        <div class="portfolio-details">
-            <img src="assets/skills-icon.svg" alt="">
-            <h4>Skills</h4>
-
-            <ul class="portfolio-languages">
-                <li>Database Management</li>
-                <li>Version Control</li>
-                <li>Cli</li>
-                <li>Web Development</li>
-                <li>Api Design</li>
-            </ul>
-        </div>
+      <div class="portfolio-details">
+        <img src="assets/icon-frameworks.svg" alt="" />
+        <h4>Frameworks</h4>
 
-        
+        <ul class="portfolio-languages">
+          <li>React.js</li>
+          <li>Ruby on Rails</li>
+          <li>RSpec</li>
+          <li>Capybara</li>
+          <li>Selenium</li>
+        </ul>
+      </div>
+
+      <div class="portfolio-details">
+        <img src="assets/skills-icon.svg" alt="" />
+        <h4>Skills</h4>
+
+        <ul class="portfolio-languages">
+          <li>Database Management</li>
+          <li>Version Control</li>
+          <li>Cli</li>
+          <li>Web Development</li>
+          <li>Api Design</li>
+        </ul>
+      </div>
+    </section>
+
+    <section id="contact-form">
+        <p>I'm always interested in hearing about new projects, so if you you'd like to chat please get in touch</p>
+        <div class="inputs">
+            <form class="contact" action="https://formspree.io/f/mgebgadd" method="post">
+                <input  type="text" id="name" type="text" placeholder="Full name" maxlength="30"   required >
+                <input  name="Email" id="email" type="email" placeholder="Email address"   required>
+                <textarea name="textarea" id="textarea" placeholder="Hey I'd like to connect to..." maxlength="500" rows="5"  required></textarea><br>
+                <button class="greenbtn" type="submit" class="greenbtn">Get in touch</button>
+            </form>
+
+            <span class="email-span">abdussalamabro@gmail.com</span>
+
+            
+        </div>
     </section>
 
+    <div id="desktop-contactform">
+      <p>I'm always interested in hearing about new projects, so if you you'd like to chat please get in touch</p>
+              <form action="https://formspree.io/f/mgebgadd" method="post">
+              <input  type="text" id="name" type="text" placeholder="Full Name" maxlength="30"   required ><br>
+              <input name="Email" id="email" type="email" placeholder="Email address"   required><br>
+              <textarea name="textarea" id="textarea" placeholder="Hey I'd like to connect to..." maxlength="500" rows="3"  required></textarea><br>
+              <button type="submit" class="greenbtn">Get in touch</button>
+          </form>   
```

Comment:
- [ ] I noticed you have two forms. Since these forms contain the same elements, why don't you use the same form and just style it differently for the mobile and desktop views? This is what the media queries are for. The two forms are showing on the mobile view.
![image](https://github.com/AbdulSalamAbro/portfoliowebsite/assets/94551484/ee9003b4-9aa1-41a6-83fa-9aa470ad84fa)

Reply:
yes I am hiding one from mobile view